HW_7/hw_7.py

```diff
@@ -0,0 +1,40 @@
+from decimal import Decimal
+import time
+from colors import bcolors
+
+
+# SECOND TASK
+class frange:
```

Comment:
`you created generator and it should be iterator with methods '__iter__' and '__next__'`